### PR TITLE
[stable/rabbitmq] Added support for advanced.config file

### PR DESCRIPTION
**Pull request by hodain**
Signed-off-by: Radek Hodain <radek.hodain@konicaminolta.cz>

#### What this PR does / why we need it:
I added support for classic style configuration advanced.config (described here: https://www.rabbitmq.com/configure.html#config-file-formats), which can be placed next to the new style config file rabbitmq.conf and allows easily config eg. LDAP which is in the new style impossible.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/rabbitmq/values.yaml [hunk 1]
```diff
@@ -129,6 +129,11 @@ rabbitmq:
   extraConfiguration: |-
     #disk_free_limit.absolute = 50MB
     #management.load_definitions = /app/load_definition.json
+  
+  ## Configuration file content: advanced configuration
+  ## Use this as additional configuraton in classic config format (Erlang term configuration format) 
+  advancedConfiguration: |-
+
```
Comment (carrodher) [inline stable/rabbitmq/values.yaml]: This line is not needed

**Review by carrodher — COMMENTED** [review]
Thanks for the contribution, before continue, you should increment the chart version in the `Chart.yml` according to [semver](http://semver.org/):

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

File: stable/rabbitmq/values.yaml (hunk 1 above)
Comment (hodain) [reply stable/rabbitmq/values.yaml]: Should I remove it or let it in place as a comment similarly as priorityClassName or existingClaim?

Comment (carrodher) [reply stable/rabbitmq/values.yaml]: mmm I was talking about the line 136, I mean, there are 2 lines between `advancedConfiguration: |-` and the following useful line while with only one line is ok

**Comment by carrodher** [comment]
Also, take a look at the CI, it seems there are some trailing spaces: https://circleci.com/gh/helm/charts/81283?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: stable/rabbitmq/values.yaml (hunk 1 above)
Comment (hodain) [reply stable/rabbitmq/values.yaml]: Should be fixed

**Pull request merged**